server/service/src/invoice/customer_return/insert/generate.rs

```diff
@@ -47,7 +54,7 @@ pub fn generate(
         name_store_id: other_party.store_id().map(|id| id.to_string()),
         store_id: store_id.to_string(),
         created_datetime: current_datetime,
-        status: InvoiceStatus::New,
+        status: status,
```

Comment:
```suggestion
        status,
```

Redundant field name since the variable is already the same as the field name